chain/helpers.go

```diff
@@ -26,17 +26,12 @@ import (
 	"github.com/berachain/beacon-kit/primitives/version"
 )
 
-// ActiveForkVersionForSlot returns the active fork version for a given slot.
-func (s spec) ActiveForkVersionForSlot(slot math.Slot) common.Version {
-	return s.ActiveForkVersionForEpoch(s.SlotToEpoch(slot))
-}
-
-// ActiveForkVersionForEpoch returns the active fork version for a given epoch.
-func (s spec) ActiveForkVersionForEpoch(epoch math.Epoch) common.Version {
-	if epoch >= s.ElectraForkEpoch() {
+// ActiveForkVersionForTimestamp returns the active fork version for a given timestamp.
+func (s spec) ActiveForkVersionForTimestamp(timestamp math.U64) common.Version {
```

Comment:
nit: can we change this to take in `timestamp uint64`? I see most places its called with a (u)int64 and even here its Unwrapped. so the U64 type feels unnecessarily introduced in this context.